app/Http/api_routes.php

```diff
@@ -22,6 +22,8 @@
         $api->resource('page', 'App\Api\V1\Controllers\PageController');
         $api->post('page/{id}/approve', 'App\Api\V1\Controllers\PageController@approve');
         $api->post('page/{id}/reject', 'App\Api\V1\Controllers\PageController@reject');
+        //Incident types routes
+        $api->resource('incidents/types', 'App\Api\V1\Controllers\IncidentTypeController', ['only' => ['show', 'update', 'store', 'destroy']]);
```

Comment:
One question here.
My impression is that this sets the endpoints as managed by the Route api, but does not set the operations permitted on each endpoint (POST, GET, etc)

Reply:
From Laravel's documentation(https://laravel.com/docs/5.2/controllers), the restrictions are these:
Verb | Path | Action | Route Name
POST | /photo | store | photo.store
GET | /photo/{photo} | show | photo.show
PUT/PATCH | /photo/{photo} | update | photo.update
DELETE | /photo/{photo} | destroy | photo.destroy

Reply:
Got it, thanks!